public/js/dashboard.js

```diff
@@ -111,6 +108,8 @@ requirejs([
                 widgetManager.trackPending(kernelFuture, $widgetSubArea.get(0), model);
             });
         });
+    }, function(err) {
+        console.error('Failed to start kernel:', err);
```

Comment:
I think open that as a separate issue. Clearly we can be doing better error handling across the board.